plugin/header/README.md

```diff
@@ -2,22 +2,24 @@
 
 ## Name
 
-*header* - modifies the header for responses.
+*header* - modifies the header for queries and responses.
 
 ## Description
 
-*header* ensures that the flags are in the desired state for responses. The modifications are made transparently for
-the client.
+*header* ensures that the flags are in the desired state for queries and responses.
+The modifications are made transparently for the client and subsequent plugins.
 
 ## Syntax
 
 ~~~
 header {
-    ACTION FLAGS...
-    ACTION FLAGS...
+    SELECTOR ACTION FLAGS...
```

Comment:
`SELECTOR` should be optional for now with `[SELECTOR]`

Reply:
Adjusted as requested.